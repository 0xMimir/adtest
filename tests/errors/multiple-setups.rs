fn main(){

}

#[adtest::adtest(setup = setup, setup = setup)]
fn invalid_test(){

}

fn setup(){

}