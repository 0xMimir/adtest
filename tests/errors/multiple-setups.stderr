error: Setup already defined
 --> tests/errors/multiple-setups.rs:5:1
  |
5 | #[adtest::adtest(setup = setup, setup = setup)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `adtest::adtest` (in Nightly builds, run with -Z macro-backtrace for more info)
